packages/trpc/server/routers/viewer/availability/team/listTeamAvailability.handler.ts

```diff
@@ -67,9 +67,7 @@ async function getTeamMembers({
     },
     cursor: cursor ? { id: cursor } : undefined,
     take: limit + 1, // We take +1 as itll be used for the next cursor
-    orderBy: {
-      id: "asc",
-    },
+    orderBy: [{ userId: "asc" }, { id: "asc" }], // Prisma require a unique field for tie breaking duplicate value for pagination
```

Comment:
why do we doing this? will it solve the UI issue ?

Reply:
Yes. 

The `userId` is used to ensure we only get one membership record per user, even if a user might have multiple memberships. 

In the current view, data is duplicated in pagination as Prisma requires an order by unique field to avoid duplicated value in pagination. One part of the code that implemented the same logic is legacyListMembers.

Cursor-based pagination requires you to sort by a sequential, unique column such as an ID or a timestamp. This value - known as a cursor - bookmarks your place in the result set and allows you to request the next set.

The same pattern can be found in legacyListMembers.handler.ts - line 89 -> 93